src/Backend/Modules/Location/Engine/Model.php

```diff
@@ -24,17 +27,21 @@ class Model
      */
     public static function delete(int $id): void
     {
-        // get database
-        $database = BackendModel::getContainer()->get('database');
+        /** @var EntityRepository $repository */
+        $repository = BackendModel::get('doctrine.orm.default_entity_manager')->getRepository(Location::class);
```

Comment:
maybe an idea to create a private static method to get this repository? Since it is a recurring piece of code